src/components/Homepage/Carousel/index.astro

```diff
@@ -0,0 +1,54 @@
+---
+import CarouselItem from "./Item.astro";
+import xkcdImage from "../../../assets/images/xkcd.svg";
+import audioGif from "../../../assets/images/audio.gif";
+import codeImage from "../../../assets/images/code.png";
+import "./Scrolling.css";
+---
+
+<div class="mt-6 md:mt-12 relative overflow-hidden w-full">
+  <div id="carousel" class="carousel-scroll flex gap-8 items-center">
+    {
+      [...Array(2)].map(() => (
+        <div class="flex gap-8 sm:gap-16 items-center min-w-max pr-4 sm:pr-8">
```

Comment:
I have a sneaky suspicion that this right here is what's causing the "jump" that does my head in.

Try defining that `<div>` outside the `Array.map()`.